rwslib/builders.py

```diff
@@ -3058,6 +3064,73 @@ def __lshift__(self, other):
         self.set_list_attribute(other, MdsolCheckAction, 'check_actions')
 
 
+class MdsolProtocolDeviation(TransactionalElement):
+    """
+    Extension for Protocol Deviations in Rave
+    
+    .. note:: This is a Medidata Rave Specific Extension
+    .. note:: This primarily exists as a mechanism for use by the Clinical Audit Record Service, but it is useful 
+        to define for the builders
+    """
+    ALLOWED_TRANSACTION_TYPES = ["Insert"]
+
+    def __init__(self, value, status, repeat_key=1, code=None, klass=None, transaction_type=None):
```

Comment:
Yeah I concur, but `cls` is aligned with classmethod, I think I'll keep as is (and I'll grin and bear it).  There's at least one other case of this in the builders.